eduNEXT/openedx-woocommerce-plugin
# feat: checkbox to enable options if product is a course

## Description

Created a checkbox that allows users to perform a 'double-check' of the product in case it's a course, enabling them to add their course_id and course mode properties.

## Testing instructions

To test it, you should access a WooCommerce product and scroll down to the product options. You will see the new checkbox, and it will be sufficient to check it, uncheck it, and save.

https://github.com/eduNEXT/openedx-woocommerce-plugin/assets/52968528/4ab70bcb-812b-4cb8-bb9e-12101527b4f4

## Additional information

A JavaScript function has been added to control this checkbox, which will always check its state and, depending on it, show or disable the options. This JavaScript is inserted using 'wp_enqueue_script,' which is a WordPress function that processes custom JavaScript files.

## Checklist for Merge
